SignalServiceKit/src/Util/OWSBackgroundTask.m

```diff
@@ -131,40 +396,38 @@ - (void)startBackgroundTask
             self.completionBlock = nil;
         }
         if (completionBlock) {
-            completionBlock(BackgroundTaskState_CouldNotStart);
+            DispatchMainThreadSafe(^{
+                completionBlock(BackgroundTaskState_CouldNotStart);
```

Comment:
We now guarantee that we always call `completionBlock` on the main thread, which we should have been doing all along.  I don't think it was an issue though.